WordPress/src/main/java/org/wordpress/android/ui/stats/StatsDeeplinkActivity.java

```diff
@@ -100,19 +104,17 @@ protected void onStart() {
     protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
         // show the post if user is returning from successful login
-        if (requestCode == RequestCodes.DO_LOGIN && resultCode == RESULT_OK) {
+        if (requestCode == RequestCodes.JETPACK_LOGIN) {
```

Comment:
Do we know for certain that the login was successful here? Also, I think the above comment is a copy & paste mistake - "show the post" doesn't apply to this situation.

Reply:
We probably don't, I'll add some "if not ok, finish"